Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -861,6 +865,11 @@ class BrowserViewController: UIViewController {
                 let progress = change?[.newKey] as? Float else { break }
             if !(webView.url?.isLocalUtility ?? false) {
                 urlBar.updateProgressBar(progress)
+                if progress > 0.9 {
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 0.200) {
```

Comment:
Please add a comment why this delay is here: " Profiler.end triggers a screenshot, and a delay is needed here to capture the correct screen (otherwise the screen prior to this step completing is captured)."